ivtest/regress-sv.list

```diff
@@ -521,11 +521,15 @@ struct_signed		normal,-g2009		ivltests
 sv-constants		normal,-g2005-sv	ivltests
 sv_ap_parray1		normal,-g2005-sv	ivltests
 sv_ap_parray2		normal,-g2005-sv	ivltests
+sv_ap_parray3		normal,-g2005-sv	ivltests
+sv_ap_parray4		normal,-g2005-sv	ivltests
```

Comment:
Can you move these tests to the new format? Check out the regress-vvp.list, and the documentation here: https://steveicarus.github.io/iverilog/developer/regression_tests.html

I'm trying to close out the old format and move things forward. If the documentation is not clear, I'm open to suggestions. (Or PR's, of course.)

Reply:
I was hoping we could just do automatic bulk conversion of all of these at once.